xbmc/powermanagement/PowerManager.cpp

```diff
@@ -256,6 +258,11 @@ void CPowerManager::OnSleep()
 
   CServiceBroker::GetPVRManager().OnSleep();
   g_application.SaveFileState(true);
+  m_lastFileItem.reset(new CFileItem(g_application.CurrentFileItem()));
+  m_lastFileItem->m_lStartOffset = (int)(g_application.GetTime() * 75);
+  CLog::Log(LOGDEBUG, "CPowerManager::OnSleep stack size %i", g_application.CurrentStack().Size());
+  //TODO m_lastFileItemList.reset(new CFileItemList(g_application.CurrentStack()))
+  CLog::Log(LOGDEBUG, "CPowerManager::OnSleep playerstate %s", g_application.m_pPlayer->GetPlayerState().c_str());
```

Comment:
there is nothing useful in GetPlayerState, drop it

Reply:
sure,
I was looking for "pause", I guess in kodi we do just look for play speed, isn't it?

Reply:
GetPlayerState is some relic, not even set for most cases